### ISSUE TITLE
Support cascading tags

### ISSUE THREAD

**Issue by tizmagik** [tizmagik/react-head#11]
A potentially useful feature is the ability to support cascading tags with the same `tag` and/or `name` attribute.

This may be just a matter of collecting tags to an associative array (object) where the key is the tag name and then overriding previously collected tags as the components render in order.

Similar functionality [exists in react-helmet](https://github.com/nfl/react-helmet#example) which is inspiration for this library to begin with

**Comment by tizmagik** [comment]
It means something like this:

```js
<div>
  <Title>Some upper title</Title>
  <div>
    <div><Title>Some nested title</Title></div>
  </div>
</div>
```

Once the rendering is done, the "winning" Title should be `Some nested title`

**Comment by tizmagik** [comment]
Yes, I believe so

**Comment by TrySound** [comment]
I checked it. Seems like it is browser behaviour. They love to make things harder.

**Comment by TrySound** [comment]
Cascading is supported for `title` and `meta` tags. Can this be closed?

**Comment by tizmagik** [comment]
Yes, I think so. Thanks @TrySound 

Closed via #39 and #37

**Comment by juan-bastidas** [comment]
Is there a way to about some Links to be duplicated, especially canonical?
I've found it repeated 3 times

<link rel="canonical" content="http://jeremygayed.com/" />

**Comment by tizmagik** [comment]
@juan-bastidas mind creating a new issue? I'm not sure what's the use case to avoid duplicate canonical links -- that's not typically a tag that needs to cascade, but I'd be happy to review your use case in more detail in a new issue.

**Comment by juan-bastidas** [comment]
Sorry, I was about to upload a demo project when I found that it was due to an error in my side, react-head is OK
